server/sonar-db-dao/src/main/java/org/sonar/db/component/BranchDto.java

```diff
@@ -147,6 +158,37 @@ public BranchDto setMergeBranchUuid(@Nullable String s) {
     return this;
   }
 
+  public BranchDto setPullRequestData(DbProjectBranches.PullRequestData pullRequestData) {
+    this.pullRequestBlob = encodePullRequestData(pullRequestData);
+    return this;
+  }
+
+  @CheckForNull
+  public DbProjectBranches.PullRequestData getPullRequestData() {
+    if (pullRequestBlob == null) {
+      return null;
+    }
+    return decodePullRequestData(pullRequestBlob);
+  }
+
+  private byte[] encodePullRequestData(DbProjectBranches.PullRequestData pullRequestData) {
+    ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+    try {
+      pullRequestData.writeTo(outputStream);
+      return outputStream.toByteArray();
+    } catch (IOException e) {
+      throw new IllegalStateException("Fail to serialize pull request data", e);
+    }
+  }
+
+  private DbProjectBranches.PullRequestData decodePullRequestData(byte[] pullRequestBlob) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make "decodePullRequestData" a "static" method. ([squid:S2325](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2325))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fjanos%2FSONAR-10365%2Fadd-pull-request-data-column&issues=AWGfmDwFjNA6aRBI4VoI&open=AWGfmDwFjNA6aRBI4VoI)

[ISSUE_KEY]: AWGfmDwFjNA6aRBI4VoI